DatadogLogs/Sources/Feature/MessageReceivers.swift

```diff
@@ -171,7 +192,7 @@ internal struct CrashLogReceiver: FeatureMessageReceiver {
             return send(report: crash.report, with: crash.context, to: core)
         } catch {
             core.telemetry
-                .error("Fails to decode crash from RUM", error: error)
+                .error("Failed to decode crash message in `LogMessageReceiver`", error: error)
```

Comment:
This is to differentiate it from [other telemetry](https://github.com/DataDog/dd-sdk-ios/blob/d51149cac0560bf947e46dd9b472dba9b97c532c/DatadogRUM/Sources/Integrations/CrashReportReceiver.swift#L146) with the same message.